src/main/java/net/masterthought/cucumber/json/Feature.java

```diff
@@ -104,9 +104,9 @@ public Status getStatus() {
     }
 
     public String getName() {
-        return StringUtils.isNotEmpty(name) ? getStatus().toHtmlClass()
-                + "<div class=\"feature-line\"><span class=\"feature-keyword\">" + keyword + ":</span> " + name
-                + "</div></div>" : "";
+        return StringUtils.isNotEmpty(name) ? getStatus().toHtmlClass("")
+                + "<div class=\"feature-line\"><span class=\"feature-keyword\">" + keyword + ":</span> " + name +
+        "<div style=\"float: right;\"><span class=\"expanded\" onclick=\"hideall();\">&nbsp;&nbsp;&nbsp;</span><span class=\"collapsed\" onclick=\"showall();\">&nbsp;&nbsp;&nbsp;</span></div></div>"+"</div>" : "";
```

Comment:
there are 2 elements - one for `hideAll();` and one for `showAll();` 
default state is open - as it was in pr #13 . I believe that most user dosn't have many scenarios in one feature and they would be annoyed to have to click showall button